__tests__/__utils__/handlers.ts

```diff
@@ -48,6 +48,21 @@ const ForDefinitions = {
       .withPayload({})
       .returns({ unrevisedEntityIds: entities.map((entity) => entity.id) })
   },
+  DeletedEntitiesQuery(entities: Model<'AbstractEntity'>[]) {
+    given('UuidQuery').for(
+      entities.map((entity) => {
+        return { ...entity, trashed: true }
+      })
+    )
+    given('DeletedEntitiesQuery').returns({
```

Comment:
```suggestion
    given('DeletedEntitiesQuery').isDefinedBy({
```

Use a short impl with `first`, `after` and `instance` here